### PR TITLE
Bundle LaTeX-Workshop with webpack

### PR DESCRIPTION
**Pull request by NavyAdmiral**
This is too heavily borrowed from the author of the other great extensions @fabiospampinato so all credits go to him. From what I'm seeing the activation times are reduced by 80%.

In order to test, just type `yarn compile` after cloning the branch.

@James-Yu @jlelong You can take a look at this now. Workflow after this PR should still be the same: `npm run compile/yarn compile`. I don't get any errors after testing and the startup times are better with less dist files.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -831,23 +832,28 @@
     "workspaceSymbolProvider": "true"
   },
   "dependencies": {
+    "bufferutil": "^4.0.0",
     "chokidar": "^2.0.3",
     "fs-extra": "^5.0.0",
+    "fsevents": "^1.2.4",
```
Comment (fabiospampinato) [inline package.json]: `fsevents` should be an optional dependency, I think it only works on macOS and its installation fails in other OSs.

File: package.json [hunk 2]
```diff
@@ -831,23 +832,28 @@
...
     "glob": "^7.1.1",
     "micromatch": "^3.0.0",
     "pdfjs-dist": "^1.8.609",
+    "ts-loader": "^5.2.1",
```
Comment (fabiospampinato) [inline package.json]: `ts-loader` is only a development dependency.

**Comment by James-Yu** [comment]
Thanks for the contribution!

**Comment by James-Yu** [comment]
Have to comment again. It's such a great improvement of the startup performance of the extension! Brilliant!

**Comment by James-Yu** [comment]
@NavyAdmiral do you mind have a check on the latest PR about hover preview? It seems that the module mathjax-node does not work with webpack, but is an essential module to the feature. Thanks in advance.

**Pull request merged**